City-of-Helsinki/drupal-helfi-platform-config
# UHF-1023 translations

How to test on platform:
- `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-UHF-1023_translations hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
- Switch to corresponding branches: 
   - `composer require drupal/helfi_api_base:dev-UHF-1023_translations && composer require drupal/helfi_platform_config:dev-UHF-1023_translations`
- Run `make new`
- Login and explore the site and check that nearly everything you see is translated to Finnish. Check f.e.
   - https://hel-platform.docker.sh/fi/admin/tools
   - https://hel-platform.docker.sh/fi/node/5/edit
   - https://hel-platform.docker.sh/fi/admin/content/media
   - and others what you can think of...
  
Also check these: 
- https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/pull/15
- https://github.com/City-of-Helsinki/drupal-helfi-platform/pull/25
- https://github.com/City-of-Helsinki/drupal-helfi/pull/85
